buildSrc/src/main/groovy/nva.nvi.rootplugin.gradle

```diff
@@ -27,6 +27,28 @@ reporting {
     }
 }
 
+// Log clickable links to console for generated test coverage reports with
+// a dedicated task that always runs (never up to date).
+gradle.projectsEvaluated {
+    tasks.register("printCoverageReportLinks") {
+        outputs.upToDateWhen { false }
+
+        // Depend on all subprojects’ coverage tasks + root coverage task
+        dependsOn(subprojects.collect { it.tasks.named("testCodeCoverageReport") })
+        dependsOn(tasks.named("testCodeCoverageReport"))
+
+        doLast {
+            def reportDirPath = "reports/jacoco/testCodeCoverageReport/html"
+            subprojects.each { subproject ->
+                def subprojectReportDir = subproject.layout.buildDirectory.dir(reportDirPath).get().asFile
+                logger.quiet("\t Coverage report for subproject: file://${subprojectReportDir}/index.html (${subproject.name})")
+            }
+            def reportDir = layout.buildDirectory.dir(reportDirPath).get().asFile
+            logger.quiet("Combined coverage report: file://${reportDir}/index.html")
+        }
+    }
+}
+
```

Comment:
This is purely a util task to make local dev easier, could perhaps be moved elsewhere